download.html

```diff
@@ -0,0 +1,51 @@
+<div xmlns="http://www.w3.org/1999/xhtml" id="download-modal" class="modal">
+
+  <style>
```

Comment:
The `<style>` element is only supposed to appear inside the `<head>` of the document. Why not move these CSS declarations to "resources/css/list_style.css" or into a dedicated file and link to it here?